policyengine_us/parameters/gov/states/me/tax/income/credits/property_tax_fairness/income_threshold.yaml

```diff
@@ -0,0 +1,63 @@
+description: Maine qualifies filers for the property tax fairness credit with income below this threshold, based on filing status and number of dependents.
+SINGLE:
+  0:
+    2021-01-01: 42_000
+    2022-01-01: 55_000
+  1:
+    2021-01-01: 42_000
+    2022-01-01: 55_000
+  2:
+    2021-01-01: 42_000
+    2022-01-01: 55_000
+HEAD_OF_HOUSEHOLD:
+  0:
+    2021-01-01: 55_000
+    2022-01-01: 70_000
+  1:
+    2021-01-01: 55_000
+    2022-01-01: 70_000
+  2:
+    2021-01-01: 68_000
+    2022-01-01: 70_000
+JOINT:
+  0:
+    2021-01-01: 55_000
+    2022-01-01: 70_000
+  1:
+    2021-01-01: 68_000
+    2022-01-01: 86_250
+  2:
+    2021-01-01: 68_000
+    2022-01-01: 86_250
+WIDOW:
+  0:
+    2021-01-01: 55_000
+    2022-01-01: 70_000
+  1:
+    2021-01-01: 68_000
+    2022-01-01: 86_250
+  2:
+    2021-01-01: 68_000
+    2022-01-01: 86_250
+SEPARATE:
```

Comment:
add a filter in the formula to exclude separate as well, given possibility of negative income